data_initialisation.py

```diff
@@ -215,16 +244,28 @@ def clean_general_expression_data(general_expression_data):
 
 def clean_specific_expression_data(specific_expression_data):
     
+    #Cleans the expression data specific to the cell line of interest by turning
+    #minus infinite strings into a floating point representation of negative
+    #infinity, duplicate genes are dropped
+    
     print("Cleaning specific expression data...")
     
-    #specific_expression_data["Specific_gene_expression"] = specific_expression_data["Specific_gene_expression"].apply(lambda expression : 0 if expression == "-Inf" else pow(2, expression))
-    specific_expression_data["Specific_gene_expression"] = specific_expression_data["Specific_gene_expression"].apply(lambda expression : 0 if expression == "-Inf" else expression)
+    #specific_expression_data = specific_expression_data.drop(specific_expression_data[specific_expression_data["Specific_gene_expression"] == "-Inf"].index)
+    #specific_expression_data["Specific_gene_expression"] = specific_expression_data["Specific_gene_expression"].apply(lambda expression : np.NINF if expression == "-Inf" else pow(2, expression))
+    specific_expression_data["Specific_gene_expression"] = specific_expression_data["Specific_gene_expression"].apply(lambda expression : np.NINF if expression == "-Inf" else expression)
     specific_expression_data = specific_expression_data.drop_duplicates(keep = False, subset = ["Gene_name"])
     
     return specific_expression_data
 
 def clean_regulatory_elements(regulatory_elements):
     
+    #Clean the regulatory elements dataframe, depending on whether it is in gff
+    #or bed file format, either by dropping elements which are not enhancers,
+    #and then dropping unecessary columns, or by removing "chr" from chromosome
+    #strings, including only elements that have flags we are interested in, then
+    #removing the flags, separately creating a quiescent dataframe in the same
+    #way
```

Comment:
This is all a little much for one function; if you are 'separately creating a quiescent dataframe', this needs to be done in a separate function. 

Also, the language is a little difficult: I'm not sure what this means past "removing chr from chromosome strings". What does 'elements' mean, for example, in 'including only elements that...'? What does 'in the same way' mean, when you say you are creating a dataframe? In the same way as what exactly?

Reply:
Rephrased a bit, "chr" refers to literal string "chr" that is appended to all chromosome values and must be removed before analysis, have left quiescent regions in for now, as it now only adds one line as apposed to a whole new function, though maybe this can be improved